platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/event/tipset/TipsetEventCreatorImpl.java

```diff
@@ -310,49 +310,14 @@ private GossipEvent buildEvent(@Nullable final EventDescriptor otherParent) {
         return event;
     }
 
-    /**
-     * Get the generation of a descriptor, handle null appropriately.
-     */
-    private static long getGeneration(@Nullable final EventDescriptor descriptor) {
-        if (descriptor == null) {
-            return GENERATION_UNDEFINED;
-        } else {
-            return descriptor.getGeneration();
-        }
-    }
-
-    /**
-     * Get the hash of a descriptor, handle null appropriately.
-     */
-    @Nullable
-    private static Hash getHash(@Nullable final EventDescriptor descriptor) {
-        if (descriptor == null) {
-            return null;
-        } else {
-            return descriptor.getHash();
-        }
-    }
-
-    /**
-     * Get the creator of a descriptor, handle null appropriately.
-     */
-    @Nullable
-    private static NodeId getCreator(@Nullable final EventDescriptor descriptor) {
-        if (descriptor == null) {
-            return CREATOR_ID_UNDEFINED;
-        } else {
-            return descriptor.getCreator();
-        }
-    }
-
     /**
      * Given the parents, build the event object.
```

Comment:
Small note: you changed the names of the two "build event" methods, but not the javadocs to indicate a distinction

Reply:
Also, changed the `buildAndProcessEvent()` javadoc to

```
Given an other parent, build the next self event and process it.
```